### PR TITLE
Passing config data from backend to frontend with Database calls

### PR DESCRIPTION
**Pull request by nicolepeverley**
This PR sets up the functionality to pass config data from the backend to the frontend and vice versa using basic database calls for hubspot sync and saasquatch sync controls.

https://trello.com/c/wxLk8eV3/60-pass-config-data-to-backend

### CODE REVIEW

File: src/database.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+//abc();
+function hashValue(stringValue: string){
+    stringValue.toLowerCase()
+    return crypto.createHash('sha1').update(stringValue).digest('hex');
+}
+
+/**
+ * Adds Values to Database
+ */
+export function AddToDatabase(email: string, data: blankPassObject) {
```
Comment (ChristopherRDunn) [inline src/database.ts]: We could probably use the Configuration type here (in types.ts) and get rid of the blankPassObject type altogether

File: frontend/src/components/ConfigurationP2.tsx [hunk 2]
```diff
@@ -126,6 +141,36 @@ export function Controller(){
   };
   
   const handleSubmit = () => {
+     // Here we will make the requests to the backend to store the config info and to begin the integration and redirect to second config screen
+     const postConfigData = async () => {
+      return await fetch(API_CONFIGURATION_URL, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({ 
+          ConnectToHubspot: false,
```
Comment (ChristopherRDunn) [inline frontend/src/components/ConfigurationP2.tsx:150]: Should this be false? It looks like it'd overwrite the config we saved on the first config page

**Review by ChristopherRDunn — COMMENTED** [review]
This looks great! I just left a couple little comments

File: frontend/src/components/ConfigurationP2.tsx (hunk 2 above)
Comment (nicolepeverley) [reply frontend/src/components/ConfigurationP2.tsx:150]: So I just put it as false for now! We'll need to keep track of the data on both pages for if a user wants to go back or forward in the flow and I don't think we have that set up yet as right now the config data is confined to its own page.

File: frontend/src/components/ConfigurationSuccess.tsx [hunk 3]
```diff
@@ -69,8 +69,8 @@ export function View(states: states){
       <PageContent>
         <TitleText>HubSpot Integration is Connected</TitleText>
         <InfoText>Select a step below to reconfigure your integration</InfoText>
-        <TitleLinkText onClick={states.handleRedirectP1}>Step 1: Configure your <Logo src={SaaSquatchLogo} /> Integration</TitleLinkText>
-        <TitleLinkText onClick={states.handleRedirectP2}>Step 2: Configure your <Logo src={HubspotLogo} /> Integration</TitleLinkText>
+        <TitleLinkText onClick={states.handleRedirectP1}>Step 1: Configure your <Logo src={HubspotLogo} /> Integration</TitleLinkText>
```
Comment (ChristopherRDunn) [inline frontend/src/components/ConfigurationSuccess.tsx:72]: Nice catch

**Review by ChristopherRDunn — APPROVED** [review]
👍

**Review by MCunningham211 — APPROVED** [review]
This looks good and appears to work nicely. The token issue that Nicole found does not seem to be something related to the changes made in this branch.

**Pull request merged**